### PR TITLE
Add NewMoney helper func

### PR DESCRIPTION
**Pull request by marcoshuck**
This PR allow us to convert `cents` to a `money.Money` protobuf message.

### CODE REVIEW

File: utility_test.go [hunk 1]
```diff
@@ -108,3 +108,49 @@ func TestNewDateTime(t *testing.T) {
 	assert.Equal(t, time.Duration(offset)*time.Second, date.GetUtcOffset().AsDuration())
 	t.Log("Offset:", offset, "Got:", date.GetUtcOffset().AsDuration().String())
 }
+
+func TestNewMoney(t *testing.T) {
```
Comment (AlejoAsd) [inline utility_test.go:112]: This might already be covered by the fuzzing tests, but please include tests for negative numbers and ensure that all requirements are met.

**Review by AlejoAsd — CHANGES_REQUESTED** [review]

File: utility_test.go (hunk 1 above)
Comment (marcoshuck) [reply utility_test.go:112]: Maybe it went unnoticed but there's already a test covering negative numbers.

File: utility.go
[the remark this answers is not in the document]
Comment (marcoshuck) [reply utility.go:1]: The `gz` import is very convenient, I don't think it would be a good idea to use `proto`as it will collide with different protobuf packages.

[the remark this answers is not in the document]
Comment (AlejoAsd) [reply utility.go:1]: Then can we at least move relevant functions to a `proto_utilities.go` file or something similar? This file is already a mixed bag of things as it is. Dumping more unrelated helper functions into it will make things difficult to browse.

File: utility_test.go (hunk 1 above)
Comment (AlejoAsd) [reply utility_test.go:112]: Yep, you're right. My quick scanning failed me. Thanks!

File: utility.go
[the remark this answers is not in the document]
Comment (marcoshuck) [reply utility.go:1]: Sure thing, let's do that.

**Review by AlejoAsd — APPROVED** [review]
